include/odp/api/spec/ipsec.h

```diff
@@ -146,6 +146,13 @@ typedef struct odp_ipsec_inbound_config_t {
 	 */
 	odp_proto_chksums_t chksums;
 
+	/** Drop TFC dummy packets (NH=59) instead of reporting it via
+	 *  @tfc_dummy_packet error bit.
+	 *
+	 *  @see tfc_dummy_packets_drop
+	 */
+	odp_bool_t tfc_dummy_packets_drop;
```

Comment:
This configuration option is a bit confusing. We need to specifically mention inline and look aside cases. It may not be configurable in many hardwares.

Reply:
Hmm. What about `tfc_dummy_packet_drop_sync/async/inline_configurable` capabilities and three flags `tfc_dummy_packet_drop_sync/async/inline`?

Reply:
We should minimize the number of configuration options at this time. I think it's not important for application to control dummy packet dropping. So, it can be implementation defined in inline. For sync/async we must defined how many packets can be expected back from each IPSEC operation. The simplest would be that each input packet produces an output packet, and it's implementation specific what the output packet contains (NONH packet or just a warning flag telling that the packet was dropped since it was NONH). Application would not need to control dropping but notice if that happened. Async interface would be problematic for application if it's possible that nothing comes back from an operation (all packets silently dropped).

Reply:
I understood that we wanted the automatic dropping only in inline processing mode and not in other modes.

Reply:
Updating to specifically mention "inline" mode. We can add drop/etc. settings to sync/async modes later.

Reply:
@psavol @JannePeltonen for async the major issue is the packet metadata/user pointer here. I think we might need to give some thought to this area anyway because of fragmentation handling. We might need additional copy_udata/free_udata function pointers to sa config.